be2-scala/docs/README.md

```diff
@@ -206,6 +206,8 @@ Summary of the keys used to retrieve data:
 - for a message: `channel#message_id`
 - for ChannelData: `channel`
 - for LaoData: `root/lao_id#laodata`
+- for RollCallData: `root/rollcall/lao_id`
+- for ElectionData: `root/private/election_id`
```

Comment:
it's not indexed by lao? 

Disclaimer: this is not a documentation problem, just an odd choice that I'd forgotten about and possibly something the students will want to improve 😅

Reply:
True that was a odd choice. Since there was no guidelines (I think) about those new objects, we simply decided it by our own. It is also easily changed for the next students.